guides/common/modules/con_how-content-view-environments-are-handled-during-registration-with-multiple-activation-keys.adoc

```diff
@@ -0,0 +1,49 @@
+[id="how-content-view-environments-are-handled-during-registration-with-multiple-activation-keys"]
+= How content view environments are handled during registration with multiple activation keys
+
+Registering a host with multiple activation keys assigns different attributes from each key, and their combined settings determine the host's configuration.
+Note that multiple activation keys and multiple content view environments are not the same.
+Activation keys assign specific attributes to hosts during registration.
+For instance, one activation key assigns system purpose attributes, another assigns repository set content overrides, and a third assigns the host's content view environments.
+When you register a host with these three activation keys, it inherits the system purpose attributes, content overrides, and content view environments from all the keys combined.
+
+When multiple activation keys assign the same attributes, {Project} decides how to resolve conflicts.
+In general, the host receives the union of non-conflicting settings and the setting from the last activation key for conflicting settings.
+
+Conflicting settings (where the setting from the last activation key wins) include:
+
+* System purpose attributes
+* Release version
+
+Non-conflicting settings (where the host receives the union of them) include:
+
+* Host collections
+
+.Behavior according to the *Allow multiple content views* setting
+Activation key behavior regarding multiple content view environments depends on the `allow_multiple_content_views` setting.
+If you enable `allow_multiple_content_views`, content view environments are treated as non-conflicting.
+If you disable `allow_multiple_content_views`, content view environments are considered conflicting.
+
+When you enable `allow_multiple_content_views` and register a host with multiple activation keys, the host receives all content view environments from each activation key, in the order they are passed.
+====
```

Comment:
The example blocks aren't properly formatted because there are newlines missing before and after the lists. Please, fix that.